packages/mars/package.json

```diff
@@ -37,7 +37,7 @@
     "lint:prettier": "prettier './{src,test}/**/*.ts'",
     "build": "yarn build:test:contracts && tsc",
     "build:test:contracts": "waffle ./test/waffle.json",
-    "test": "mocha",
+    "test": "nyc mocha",
```

Comment:
Let's split tests with coverage and ones without. We can only run covered tests on CI but probably don't want it while developing